solitude/settings/base.py

```diff
@@ -19,14 +19,6 @@
 #
 ALLOWED_HOSTS = []
 
-if os.environ.get('MEMCACHE_URL'):
-    CACHES = {
-        'default': {
-            'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-            'LOCATION': os.getenv('MEMCACHE_URL'),
-        }
-    }
-
```

Comment:
Can you point where you looked please? I couldn't find anything else referring to memcache for solitude. I found that stage does not use it.